.github/actions/with-k-docker/Dockerfile

```diff
@@ -1,19 +1,26 @@
-FROM ubuntu:jammy
+ARG BASE_OS
```

Comment:
Do we use this image with any `BASE_OS` other than `ubuntu`? If not, consider keeping this value concrete for simplicity.